lib/src/main/java/com/thenewboston/api/bankapi/datasource/BankDataSource.kt

```diff
@@ -318,4 +319,25 @@ class BankDataSource @Inject constructor(private val networkClient: NetworkClien
             else -> Outcome.Success(response)
         }
     }
+
+    suspend fun sendClean(request: PostCleanRequest): Outcome<Clean> = makeApiCall(
+        call = { doSendClean(request) },
+        errorMessage = "An error occurred while sending the clean request"
+    )
+
+    private suspend fun doSendClean(request: PostCleanRequest): Outcome<Clean> {
+        val response = networkClient.defaultClient.patch<Clean> {
```

Comment:
Should this not be `.post `instead of `.patch` ?

Docs indicates this is a POST request. https://thenewboston.com/bank-api/clean

Reply:
Sorry my bad, will fix it